curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/use-the-u-tag-to-underline-text.chinese.md

```diff
@@ -28,8 +28,6 @@ tests:
     testString: assert($('u').length === 1, '你应该有一个 <code>u</code> 标签。');
   - text: '<code>u</code> 标签应该包围 “理工博士”。'
```

Comment:
@scissorsneedfoodtoo @raisedadead I just noticed "Ph.D. students" should have been wrapped in `code` tags in the English version, because it should not be translated.  My guess is there are many such instances.  I think we will need to search the English curriculum to find all quoted (single or double) values  used inside a test's `text` and change all applicable instances to be wrapped in `code` tags in a future PR.  For now, we just need to leave it as is.  Once I get the GitHub Actions or Probot working as we want, when we update the English version, a comment notifying the Chinese team will be automatically generated with any changes to the English challenges after the Chinese curriculum.

Reply:
Once the unified test suite is up, challenges like this in Chinese will still use and test for the "Ph.D. students" string, is that right? Will it be possible for the different language teams to translate those strings later?

Reply:
The whole point of PR #38040 is to be able to use the seed code, solutions and `text` of the Tests from the English file across all non-English files.  This removes duplication and keeps things much more organized.